def printErro(erro: Exception):
    print('Ocorreu um erro:')
    print('Detalhes:')
    print(erro)